### ISSUE TITLE
Cannot play youtube audio: EDL specifies no segments.'

### ISSUE THREAD

**Issue by sourav-majumdar-math** [mpv-player/mpv#11426]
### Important Information

mpv 0.35.1

Arch Linux

### Reproduction steps

mpv --ytdl-format="bestaudio" https://www.youtube.com/watch?v=tAGnKpE4NCI

### Expected behavior

Play audio of the youtube video

### Actual behavior
Get error

```
EDL specifies no segments.'
EDL parsing failed.
Error in EDL.
No video or audio streams selected.
```
### Log file

```
[cplayer] Command line options: '-v'
[cplayer] mpv 0.35.1 Copyright © 2000-2023 mpv/MPlayer/mplayer2 projects
[cplayer]  built on Mon Feb 20 07:38:49 2023
[cplayer] FFmpeg library versions:
[cplayer]    libavutil       57.28.100
[cplayer]    libavcodec      59.37.100
[cplayer]    libavformat     59.27.100
[cplayer]    libswscale      6.7.100
[cplayer]    libavfilter     8.44.100
[cplayer]    libswresample   4.7.100
[cplayer] FFmpeg version: n5.1.2
[cplayer]
[cplayer] Configuration: meson configure build -Dprefix=/usr -Dbuildtype=plain -Doptimization=plain
[cplayer] List of enabled features: alsa av-channel-layout build-date cdda cplayer cplugins cuda-hwaccel cuda-interop dmabuf-interop-gl dmabuf-interop-pl dmabuf-wayland drm drm-is-kms dvbin dvdnav egl egl-drm egl-helpers egl-x11 ffmpeg ffnvcodec gbm gl gl-wayland glibc-thread-name glob glob-posix gpl iconv jack javascript jpeg jpegxl lcms2 libarchive libass libavdevice libbluray libdl libm libmpv-shared libplacebo libplacebo-next librt linux-fstatfs luajit manpage-build memfd_create noexecstack pipewire plain-gl posix pulse rubberband rubberband-3 shaderc stdatomic threads uchardet vaapi vaapi-drm vaapi-egl vaapi-libplacebo vaapi-wayland vaapi-x-egl vaapi-x11 vdpau vector vt.h vulkan wayland wayland_protocols_1_24 x11 xv zimg zlib
[cplayer] Reading config file /etc/mpv/encoding-profiles.conf
[cplayer] Applying profile 'default'...
[cplayer] Setting option 'v' = '' (flags = 8)
[cplayer] mpv 0.35.1 Copyright © 2000-2023 mpv/MPlayer/mplayer2 projects
[cplayer]  built on Mon Feb 20 07:38:49 2023
[cplayer] FFmpeg library versions:
[cplayer]    libavutil       57.28.100
[cplayer]    libavcodec      59.37.100
[cplayer]    libavformat     59.27.100
[cplayer]    libswscale      6.7.100
[cplayer]    libavfilter     8.44.100
[cplayer]    libswresample   4.7.100
[cplayer] FFmpeg version: n5.1.2
[cplayer]
[cplayer] Usage:   mpv [options] [url|path/]filename
[cplayer]
[cplayer] Basic options:
[cplayer]  --start=<time>    seek to given (percent, seconds, or hh:mm:ss) position
[cplayer]  --no-audio        do not play sound
[cplayer]  --no-video        do not play video
[cplayer]  --fs              fullscreen playback
[cplayer]  --sub-file=<file> specify subtitle file to use
[cplayer]  --playlist=<file> specify playlist file
[cplayer]
[cplayer]  --list-options    list all mpv options
[cplayer]  --h=<string>      print options which contain the given string in their name
[cplayer]
[cplayer] Set property: shared-script-properties -> 1
[cplayer] Set property: shared-script-properties -> 1
[cplayer] Set property: shared-script-properties -> 1
(ins)sourav@istina:~$ mpv -v --ytdl-format="bestaudio" https://www.youtube.com/watch?v=tAGnKpE4NCI
[cplayer] Command line options: '-v' '--ytdl-format=bestaudio' 'https://www.youtube.com/watch?v=tAGnKpE4NCI'
[cplayer] mpv 0.35.1 Copyright © 2000-2023 mpv/MPlayer/mplayer2 projects
[cplayer]  built on Mon Feb 20 07:38:49 2023
[cplayer] FFmpeg library versions:
[cplayer]    libavutil       57.28.100
[cplayer]    libavcodec      59.37.100
[cplayer]    libavformat     59.27.100
[cplayer]    libswscale      6.7.100
[cplayer]    libavfilter     8.44.100
[cplayer]    libswresample   4.7.100
[cplayer] FFmpeg version: n5.1.2
[cplayer]
[cplayer] Configuration: meson configure build -Dprefix=/usr -Dbuildtype=plain -Doptimization=plain
[cplayer] List of enabled features: alsa av-channel-layout build-date cdda cplayer cplugins cuda-hwaccel cuda-interop dmabuf-interop-gl dmabuf-interop-pl dmabuf-wayland drm drm-is-kms dvbin dvdnav egl egl-drm egl-helpers egl-x11 ffmpeg ffnvcodec gbm gl gl-wayland glibc-thread-name glob glob-posix gpl iconv jack javascript jpeg jpegxl lcms2 libarchive libass libavdevice libbluray libdl libm libmpv-shared libplacebo libplacebo-next librt linux-fstatfs luajit manpage-build memfd_create noexecstack pipewire plain-gl posix pulse rubberband rubberband-3 shaderc stdatomic threads uchardet vaapi vaapi-drm vaapi-egl vaapi-libplacebo vaapi-wayland vaapi-x-egl vaapi-x11 vdpau vector vt.h vulkan wayland wayland_protocols_1_24 x11 xv zimg zlib
[cplayer] Reading config file /etc/mpv/encoding-profiles.conf
[cplayer] Applying profile 'default'...
[cplayer] Setting option 'v' = '' (flags = 8)
[cplayer] Setting option 'ytdl-format' = 'bestaudio' (flags = 8)
[cplayer] Waiting for scripts...
[cplayer] Set property: shared-script-properties -> 1
[cplayer] Set property: shared-script-properties -> 1
[cplayer] Set property: shared-script-properties -> 1
[cplayer] Set property: shared-script-properties -> 1
[osd/libass] libass API version: 0x1701000
[osd/libass] libass source: commit: 0.17.1-0-ge8ad72accd3a84268275a9385beb701c9284e5b3-dirty
[osd/libass] Shaper: FriBidi 1.0.12 (SIMPLE) HarfBuzz-ng 7.1.0 (COMPLEX)
[osd/libass] Setting up fonts...
[osd/libass] Using font provider fontconfig
[osd/libass] Done.
[osd/libass] libass API version: 0x1701000
[osd/libass] libass source: commit: 0.17.1-0-ge8ad72accd3a84268275a9385beb701c9284e5b3-dirty
[osd/libass] Shaper: FriBidi 1.0.12 (SIMPLE) HarfBuzz-ng 7.1.0 (COMPLEX)
[osd/libass] Setting up fonts...
[osd/libass] Using font provider fontconfig
[osd/libass] Done.
[cplayer] Done loading scripts.
[cplayer] Running hook: ytdl_hook/on_load
[ytdl_hook] ytdl:// hook
[ytdl_hook] not a ytdl:// url
[ffmpeg] Opening https://www.youtube.com/watch?v=tAGnKpE4NCI
[cplayer] Set property: shared-script-properties -> 1
[ffmpeg] Mime-type: 'text/html; charset=utf-8'
[demux] Trying demuxers for level=normal.
[lavf] No format found, try lowering probescore or forcing the format.
[demux] Trying demuxers for level=unsafe.
[lavf] No format found, try lowering probescore or forcing the format.
[cplayer] Opening failed or was aborted: https://www.youtube.com/watch?v=tAGnKpE4NCI
[cplayer] Running hook: ytdl_hook/on_load_fail
[ytdl_hook] full hook
[ytdl_hook] No youtube-dl found with path yt-dlp in config directories
[ytdl_hook] Found youtube-dl with path yt-dlp in PATH
[ytdl_hook] youtube-dl succeeded!
[cplayer] Set property: file-local-options/user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.41 Safari/537.36" -> 1
[ytdl_hook] format selection: youtube-dl (separate)
[cplayer] Set property: stream-open-filename="edl://!mp4_dash,init=%967%https://rr8---sn-gwpa-5hqz.googlevideo.com/videoplayback?expire=1678267722&ei=6gAIZMb0DZaSvQSTpYeACA&ip=2409%3A4041%3A2cc3%3Aa494%3Ab42%3Aa413%3A155e%3Aaaf8&id=o-AOoZFmMGjkX6u6rMoCO6SGDuRu7M15YBBATTu2pWC7Z0&itag=251&source=youtube&requiressl=yes&mh=od&mm=31%2C29&mn=sn-gwpa-5hqz%2Csn-gwpa-cvhd&ms=au%2Crdu&mv=m&mvi=8&pcm2cms=yes&pl=41&initcwndbps=305000&spc=H3gIhhfFfP-Y-xrcVsfmsP2kgojVsWI&vprv=1&svpuc=1&mime=audio%2Fwebm&gir=yes&clen=6639404&dur=385.741&lmt=1661874493540825&mt=1678245709&fvip=4&keepalive=yes&fexp=24007246&c=ANDROID&txp=4532434&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRgIhAPAesTA7CbZGdWv38kb3EISPpG9ImidxJdizAenR2UmZAiEA7zgJKAZUepAhNCpi0luYSxUaB0G4WLtGDOlpY-9dqME%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpcm2cms%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIgcS8cXhCsG3Lq8rodYzc6DKJYZx7FQT_MNBO4c3-cEPYCIQCh2t3fxNOPyhHe7wAHsShbR853YXoh__8mq3qmQEHxAg%3D%3D&range=0-6639404;" -> 1
[cplayer] Set property: file-local-options/force-media-title="Metallica: Nothing Else Matters (Official Music Video)" -> 1
[cplayer] Set property: file-local-options/stream-lavf-o={} -> 1
[edl] Opening edl://!mp4_dash,init=%967%https://rr8---sn-gwpa-5hqz.googlevideo.com/videoplayback?expire=1678267722&ei=6gAIZMb0DZaSvQSTpYeACA&ip=2409%3A4041%3A2cc3%3Aa494%3Ab42%3Aa413%3A155e%3Aaaf8&id=o-AOoZFmMGjkX6u6rMoCO6SGDuRu7M15YBBATTu2pWC7Z0&itag=251&source=youtube&requiressl=yes&mh=od&mm=31%2C29&mn=sn-gwpa-5hqz%2Csn-gwpa-cvhd&ms=au%2Crdu&mv=m&mvi=8&pcm2cms=yes&pl=41&initcwndbps=305000&spc=H3gIhhfFfP-Y-xrcVsfmsP2kgojVsWI&vprv=1&svpuc=1&mime=audio%2Fwebm&gir=yes&clen=6639404&dur=385.741&lmt=1661874493540825&mt=1678245709&fvip=4&keepalive=yes&fexp=24007246&c=ANDROID&txp=4532434&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRgIhAPAesTA7CbZGdWv38kb3EISPpG9ImidxJdizAenR2UmZAiEA7zgJKAZUepAhNCpi0luYSxUaB0G4WLtGDOlpY-9dqME%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpcm2cms%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIgcS8cXhCsG3Lq8rodYzc6DKJYZx7FQT_MNBO4c3-cEPYCIQCh2t3fxNOPyhHe7wAHsShbR853YXoh__8mq3qmQEHxAg%3D%3D&range=0-6639404;
[demux] Trying demuxers for level=request.
[demux] Detected file format: Edit decision list
[demux] Stream is not seekable.
[demux] EDL specifies no segments.'
[demux] EDL parsing failed.
[demux] Error in EDL.
[cplayer] Opening done: edl://!mp4_dash,init=%967%https://rr8---sn-gwpa-5hqz.googlevideo.com/videoplayback?expire=1678267722&ei=6gAIZMb0DZaSvQSTpYeACA&ip=2409%3A4041%3A2cc3%3Aa494%3Ab42%3Aa413%3A155e%3Aaaf8&id=o-AOoZFmMGjkX6u6rMoCO6SGDuRu7M15YBBATTu2pWC7Z0&itag=251&source=youtube&requiressl=yes&mh=od&mm=31%2C29&mn=sn-gwpa-5hqz%2Csn-gwpa-cvhd&ms=au%2Crdu&mv=m&mvi=8&pcm2cms=yes&pl=41&initcwndbps=305000&spc=H3gIhhfFfP-Y-xrcVsfmsP2kgojVsWI&vprv=1&svpuc=1&mime=audio%2Fwebm&gir=yes&clen=6639404&dur=385.741&lmt=1661874493540825&mt=1678245709&fvip=4&keepalive=yes&fexp=24007246&c=ANDROID&txp=4532434&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRgIhAPAesTA7CbZGdWv38kb3EISPpG9ImidxJdizAenR2UmZAiEA7zgJKAZUepAhNCpi0luYSxUaB0G4WLtGDOlpY-9dqME%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpcm2cms%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIgcS8cXhCsG3Lq8rodYzc6DKJYZx7FQT_MNBO4c3-cEPYCIQCh2t3fxNOPyhHe7wAHsShbR853YXoh__8mq3qmQEHxAg%3D%3D&range=0-6639404;
[cplayer] Running hook: ytdl_hook/on_preloaded
[cplayer] No video or audio streams selected.
[cplayer] finished playback, no audio or video data played (reason 4)
[cplayer]
[cplayer] Exiting... (Errors when loading file)
[cplayer] Set property: shared-script-properties -> 1
```

**Comment by christoph-heinrich** [comment]
That got fixed by 94c189dae76ba280d9883b16346c3dfb9720687e

**Comment by Dudemanguy** [comment]
Duplicate of #11392.
